web/assets/scss/pro-hub.scss

```diff
@@ -168,5 +168,8 @@
   }
 }
 
-
-
+assurance-visit {
```

Comment:
We shouldn't be duplicating code between the two hub views. Any styles that are the same in both should be in a shared .scss file.